tomland.cabal

```diff
@@ -137,7 +139,13 @@ benchmark tomland-benchmark
                       -fhide-source-paths
                       -Wmissing-export-lists
                       -Wpartial-fields
+
   build-depends:      base
+                    , aeson
+                    , deepseq <= 1.4.3.0
+                    , gauge
+                    , htoml <= 1.0.0.3
```

Comment:
It's usually better to specify with upper bound as well. Try to replace `<=` with `^>=`